gatsby-config.js

```diff
@@ -6,6 +6,14 @@ const siteUrl = 'https://docs.newrelic.com';
 
 const dataDictionaryPath = `${__dirname}/src/data-dictionary`;
 
+const autoLinkHeaders = {
```

Comment:
I'm a fan of this. Maybe we can start breaking up our crazy big config file using this pattern.